source/concepts/what-is-ember-js.md

```diff
@@ -124,15 +124,18 @@ You can tell Ember about these dependencies like this:
 
 ```javascript
 var president = Ember.Object.create({
-  firstName: "Barack",
-  lastName: "Obama",
+  init: function() {
+    this._super.apply(this, arguments);
+    this.set('firstName', 'Barack');
```

Comment:
and wouldn't `Ember.on('init', function() {` be favoured so the user doesn't have to call super?